leader-election-impl/src/main/java/com/palantir/leader/proxy/AwaitingLeadershipProxy.java

```diff
@@ -100,6 +107,7 @@ private void tryToGainLeadership() {
 
     private void gainLeadership() {
         try {
+            leaderLog.info("Not the leader. Going to sleep.");
```

Comment:
this should probably be `DEBUG` or `TRACE` or you'll have some unhappy folks paging you that disks are filled